odkx-src/sync-endpoint-manual-setup.rst

```diff
@@ -197,3 +197,23 @@ Stopping ODK-X Sync Endpoint
 
       $ docker volume rm (docker volume ls -f "label=com.docker.stack.namespace=syncldap" -q)
 
+.. _sync-anonymous:
+
+Anonymous Access for ODK-X Sync Endpoint
+-----------------------------------------
+
+Navigate to :file:`sync-endpoint-default-setup/config/sync-endpoint/security.properties` 
+
+  - To Enable Anonymous access set the following fields to *true*
+
+    .. code-block::
+
+      sync.preference.anonymousTablesSync=true
+      sync.preference.anonymousAttachmentAccess=true
+
+  - To Disable Anonymous access set the following fields to *false*
```

Comment:
Could you add a few words on what to do after changing the configuration file on an existing instance? For example, when using the default Docker Stack setup, the Docker Config needs to be updated by either recreating the Docker Config or redeploying the Docker Stack.